### ISSUE TITLE
EDIT-EXT not showing

### ISSUE THREAD

**Issue by Omaniac** [thaider/Tweeki#121]
In tweeki-sidebar-right I use EDIT-EXT, but it only shows the default EDIT button. Any ideas why?

MW: 1.31.0
PHP: 7.0.30
Tweeki: 1.10

**Comment by thaider** [comment]
Yes, that's exactly the page. „Show advanced features“ should be somewhere at the bottom of „Utseende“/„Appearance“.
